GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1311,8 +1338,7 @@ bool AddArtificialRevisions(IReadOnlyList<ObjectId>? headParents = null)
                 if (headParents is null)
                 {
                     // Add as normal commits at current position
-                    _gridView.Add(workTreeRev);
-                    _gridView.Add(indexRev);
+                    _gridView.AddRange([workTreeRev, indexRev]);
```

Comment:
```suggestion
                    revisionsToDisplay.AddRange([workTreeRev, indexRev]);
```

Reply:
I prefer not doing that because otherwise I really don't know how to convert the code of the `else`.